### PR TITLE
install code formatter

### PR DESCRIPTION
**Pull request by sh0hei**
Formatting with [sbt-scalariform](https://github.com/sbt/sbt-scalariform)

### CODE REVIEW

**Comment by xuwei-k** [comment]
I recommend check formatting with travis-ci
- https://github.com/scalikejdbc/scalikejdbc/blob/15ac3cd80eafa7bc7376bd8d61a6c2c534832e2a/project/Build.scala#L103-L108
- https://github.com/sbt/sbt/pull/2676

File: src/main/scala/jp/ne/opt/chronoscala/Interval.scala [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package jp.ne.opt.chronoscala
 
-import java.time.{ZonedDateTime, Duration, ZoneId, Instant}
+import java.time.{ ZonedDateTime, Duration, ZoneId, Instant }
```
Comment (ocadaruma) [inline src/main/scala/jp/ne/opt/chronoscala/Interval.scala]: [IMO] I prefer to disable `SpacesAroundMultiImports`.

File: src/test/scala/jp/ne/opt/chronoscala/RichZonedDateTimeSpec.scala [hunk 2]
```diff
@@ -10,11 +10,12 @@ object RichZonedDateTimeSpec extends Properties("RichZonedDateTime") with Gens {
     a <- zonedDateTimeGen
     b <- zonedDateTimeGen
     c <- zonedDateTimeGen
-  } yield (a, b, c)) { case (a, b, c) =>
-    val antisymmetry = !(a <= b && b <= a) || a == b
-    val transitivity = !(a <= b && b <= c) || a <= c
-    val totality = a <= b || b <= a
+  } yield (a, b, c)) {
+    case (a, b, c) =>
```
Comment (ocadaruma) [inline src/test/scala/jp/ne/opt/chronoscala/RichZonedDateTimeSpec.scala]: I prefer `yield` and `case (a, b, c) =>` in one line.
(I don't know there is such preference or not)

**Comment by ocadaruma** [comment]
Thanks

**Pull request merged**